CONTRIBUTING.md

```diff
@@ -331,93 +330,6 @@ satisfy the tests provided for each problem.
 You can include advice and helpful links for your language track in the
 `SETUP.md` file.
 
-## Maintaining a Track
-
-Maintaining a language track generally consists of:
-
-- Reviewing/merging pull requests.
-- Discussing improvements in the exercises.
-- Implementing or porting new exercises.
-- Improving the development tooling (e.g. implementing continuous
-  integration).
-- Language-Specific support.
-- Adding/improving language-specific documentation.
-
-Ideally a track will have several maintainers, for two reasons:
```

Comment:
This section belongs in the documentation about requesting a new language track. I've added it to the issue here: https://github.com/exercism/docs/issues/21